### DISCUSSION TITLE
Unclear how to import from KOReader

### DISCUSSION THREAD

**Discussion by ezpzalexander** [FreeLanguageTools/vocabsieve#44] [General]
How to import from KOReader?
What file format should it be? HTML, plain text, ..;?

When I use KOReader, the export gets in D:\.adds\koreader\clipboard

Which folder should I select?
![image](https://user-images.githubusercontent.com/42479112/213873106-dd88e836-ba18-4a0f-994f-b56e0a534643.png)

**Comment by 1over137** [comment]
Select a folder containing *all* the ebooks you want to import. It reads your highlights data directly (in bookname.sdr folders), not your KOReader databases. You don't ever need to export anything from KOReader. Only epub and fb2 books are supported.

**Reply by ezpzalexander** [reply]
This is the error I get when I select the folder that contains all the ebooks (.sdr)
What can I do?
![image](https://user-images.githubusercontent.com/42479112/213913522-24adbbdd-2431-48fd-91ea-9ef349373b79.png)

**Reply by 1over137** [reply]
Seems like I accidentally closed the tab before I could reply. You should try to separate out the ebook causing the bug (by copying all to a separate folder, and remove some to see which one is causing the issue) and using the rest of the books in the mean time. Consider sending me that ebook so I can try and fix this.
